### PR TITLE
AC-967 Resolved NPE in Register Patient Activity

### PR DESCRIPTION
**Pull request by shivamsawlani02**
## Description of what I changed
Updated the older version of the chucker library with the newer version. This resolved NPE in Register Patient Activity.

## Issue I worked on
JIRA Issue: https://issues.openmrs.org/browse/AC-967

## Checklist: I completed these to help reviewers :)
- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by shivamsawlani02** [comment]
@LuGO0 @rishabh-997 please review.

File: openmrs-client/build.gradle [hunk 1]
```diff
@@ -157,10 +157,9 @@ dependencies {
     // Work Manager
     implementation "androidx.work:work-runtime:${versions.workManager}"
 
-    // http request Chuck Library
-    testImplementation "com.readystatesoftware.chuck:library:${versions.chuckLibrary}"
-    debugImplementation "com.readystatesoftware.chuck:library:${versions.chuckLibrary}"
-    releaseImplementation "com.readystatesoftware.chuck:library-no-op:${versions.chuckLibrary}"
+    // http request Chucker Library
+    debugImplementation "com.github.chuckerteam.chucker:library:${versions.chuckerLibrary}"
+    releaseImplementation "com.github.chuckerteam.chucker:library-no-op:${versions.chuckerLibrary}"
```
Comment (LuGO0) [inline openmrs-client/build.gradle:162]: @shivamsawlani02 do we need a testImplementation as well?

**Review by LuGO0 — COMMENTED** [review]
Just one thing  rest lGTM

File: openmrs-client/build.gradle (hunk 1 above)
Comment (shivamsawlani02) [reply openmrs-client/build.gradle:162]: > @shivamsawlani02 do we need a testImplementation as well?

@LuGO0 It was not there in the official documentation of chucker, so I didn't include it. Should I include that too?

**Comment by LuGO0** [comment]
@rishabh-997 would you take a look at it? It LGTM

**Comment by LuGO0** [comment]
Works for me @rishabh-997 is having interviews so I will merge this.

**Pull request merged**